src/site/_includes/components/ArticleNavigation.js

```diff
@@ -28,10 +28,15 @@ const md = require("markdown-it")();
  * @return {Array} An array of pathItem slugs.
  */
 function getPathItemsFromTopics(topics) {
-  return topics.reduce(
-    (pathItems, topic) => pathItems.concat(topic.pathItems),
-    [],
-  );
+  const temp = topics.reduce((reduced, topic) => {
```

Comment:
maybe this var should be named `pathItems`?

Reply:
This just shouldn't have been there 😅